app/src/controllers/uniqueErrors/sagas.js

```diff
@@ -55,18 +68,44 @@ function* fetchClusters(payload = {}) {
   } else {
     yield call(fetchParentLaunch, { payload: { project, launchId } });
   }
-  parentLaunch = yield select(launchSelector);
+
   const namespace = yield select(namespaceSelector);
   const query = yield select(queryParametersSelector, namespace);
-  yield put(
-    fetchDataAction(NAMESPACE)(
-      URLS.clusterByLaunchId(project, launchId, {
-        [PAGE_KEY]: query[PAGE_KEY],
-        [SIZE_KEY]: query[SIZE_KEY],
-        [SORTING_KEY]: query[SORTING_KEY],
-      }),
-    ),
+  const supportedIntegration = integrations.find(
+    (integration) =>
+      integration &&
+      integration.integrationType &&
+      integration.integrationType.details &&
+      integration.integrationType.details.metadata &&
+      integration.integrationType.details.metadata.supportedFeatures &&
+      integration.integrationType.details.metadata.supportedFeatures.includes('newErrors'),
```

Comment:
As `newErrors` is a part of plugin, I think make sense to rename to smth like `uniqueErrors` or `uniqueErrorsClusters`.